### ISSUE TITLE
Standardize input filename pattern

### ISSUE THREAD

**Issue by Peter9192** [WAM2layers/WAM2layers#117]
If we want to be more flexible to different types of input data, it would be helpful to have a standardized data loading function. This function needs to know about the organization of your data on your disk. One option would be to have this fixed. Alternatively, we can make it flexible by passing in a filename pattern. We suggest the following default filename pattern:

```
filename_pattern: "{ERA5}/{Year}/{Month}/{ERA5}_{Year}_{Month}_{Day}_{Variable}{levttype}.nc"  # levtype = "_ml" or "_pl" or ""
```

**Comment by Imme1992** [comment]
We have implemented this! And made a related download script for ERA5 so the data is directly saved in the right format

**Comment by Peter9192** [comment]
Closed in #147
